### PR TITLE
Finalize Bulgarian localizaiton & clean up localization-related code

### PR DESCRIPTION
**Pull request by sibir-ine**
I set up the rest of the Bulgarian localization as pointed out in https://github.com/loot/loot/issues/1434#issuecomment-798826463. @RacerBG, would you mind translating this [line](https://github.com/loot/loot/blob/5dbd5ad51f1a964da2df33d70e96148d09cfb11c/scripts/installer.iss#L188)?

### CODE REVIEW

**Comment by sibir-ine** [comment]
Ah, looks like Bulgarian was only made an official language very recently. We'll have to treat it as an unofficial language until Inno Setup's next update & until we update to it.

**Comment by RacerBG** [comment]
@sibir-ine The line you requested is translated as "Искате ли да изтриете Вашите настройки и потребителските метаданни? " without the quotes of course. :smiley:

**Comment by sibir-ine** [comment]
> 
> 
> The line you requested is translated as "Искате ли да изтриете Вашите настройки и потребителските метаданни? " without the quotes of course. 😃

@RacerBG Thank you so much! While you're here, would you be willing to take a look at translating the masterlist messages as well? If you join the [Discord server](https://loot.github.io/discord/), we have a dedicated translations channel if you need any help. Also, @pStyl3 has been working on a spreadsheet that they've released there that expedites the process significantly. We'd especially appreciate a translation for the message in loot/skyrimse#1280.

**Comment by RacerBG** [comment]
> 
> 
> > The line you requested is translated as "Искате ли да изтриете Вашите настройки и потребителските метаданни? " without the quotes of course. smiley
> 
> @RacerBG Thank you so much! While you're here, would you be willing to take a look at translating the masterlist messages as well? If you join the [Discord server](https://loot.github.io/discord/), we have a dedicated translations channel if you need any help. Also, @pStyl3 has been working on a spreadsheet that they've released there that expedites the process significantly. We'd especially appreciate a translation for the message in [loot/skyrimse#1280](https://github.com/loot/skyrimse/issues/1280).

I don't have a Discord account. Could you, please, tell me which messages I should translate?

**Comment by sibir-ine** [comment]
> 
> 
> I don't have a Discord account. Could you, please, tell me which messages I should translate?

@RacerBG There are loot/skyrimse#646 & loot/skyrimse#1280, but they are included within @pStyl3's much more complete [localizations spreadsheet](https://github.com/loot/loot/files/6150368/Masterlist_Localizations_v5.xlsx). You can include your translations in a few pull requests or provide us with a filled-out spreadsheet, whichever works best for you. Thank you so much for your contributions!

**Comment by pStyl3** [comment]
That being said, I'm gonna finish v6 of my Excel document this evening, and the included additions and changes reflect some refactored messages from the last two days. I could email you the updated version once I'm done, if you prefer it that way @RacerBG (in this case I would need your email addresse). The discord alternative is also given, where each iteration is provided.

I plan to create an online version of the document in the near future, which we then can link to, though we are not there yet.

**Comment by RacerBG** [comment]
@sibir-ine @pStyl3 I will leave comments at the appropriate bug reports but I will need some time if that's okay with both of you.

**Comment by sibir-ine** [comment]
> 
> 
> I will leave comments at the appropriate bug reports but I will need some time if that's okay with both of you.

Absolutely, take as much time as you need. I'd also recommend waiting for v6 of the spreadsheet anyway. It'll reduce the workload a bit.

**Comment by RacerBG** [comment]
> > I will leave comments at the appropriate bug reports but I will need some time if that's okay with both of you.
> 
> Absolutely, take as much time as you need. I'd also recommend waiting for v6 of the spreadsheet anyway. It'll reduce the workload a bit.

Done! :smiley:

**Review by Ortham — APPROVED** [review]

File: scripts/build_installer.ps1 [hunk 1]
```diff
@@ -9,11 +9,12 @@ function DownloadLanguageFile($languageFile, $innoPath) {
 
 $innoInstallPath = 'C:\Program Files (x86)\Inno Setup 6'
 
-# Install the unofficial Korean, Simplified Chinese and Swedish translation
-# files for Inno Setup.
+# Install the unofficial Bulgarian, Korean, Swedish and Simplified Chinese
+# translation files for Inno Setup.
+DownloadLanguageFile '\Bulgarian.isl'  $innoInstallPath
```
Comment (sibir-ine) [inline scripts/build_installer.ps1:14]: @Ortham Should adding this line also be discussed in CONTRIBUTING.md for adding unofficial translations?

Comment (Ortham) [reply scripts/build_installer.ps1:14]: Yes, I think so.

**Pull request merged**